cromshell

```diff
@@ -862,7 +859,7 @@ function execution-status-count()
 
     assertCanCommunicateWithServer "${WORKFLOW_SERVER_URL}"
     local tempFile=$( makeTemp )
-    if ! curl --connect-timeout "$CURL_CONNECT_TIMEOUT" --max-time "$CURL_MAX_TIMEOUT" --compressed -s "${WORKFLOW_SERVER_URL}/api/workflows/v1/${WORKFLOW_ID}/metadata?$CROMWELL_SLIM_METADATA_PARAMETERS" > "${tempFile}" ; then
+    if ! curl --compressed -s "${WORKFLOW_SERVER_URL}/api/workflows/v1/${WORKFLOW_ID}/metadata?$CROMWELL_SLIM_METADATA_PARAMETERS" > "${tempFile}" ; then
```

Comment:
To make this consistent with the rest of the script I think weshould break it out into 2 statements - one that performs the `curl` query and one that checks the result.